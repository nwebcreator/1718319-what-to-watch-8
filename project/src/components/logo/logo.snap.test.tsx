import { render } from '@testing-library/react';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import Logo from './logo';

describe('Component: Logo', () => {
  it('should render correctly', () => {
    const history = createMemoryHistory();
    const { container } = render(
      <Router history={history}>
        <Logo />
      </Router>,
    );
    expect(container).toMatchSnapshot();
  });
});
