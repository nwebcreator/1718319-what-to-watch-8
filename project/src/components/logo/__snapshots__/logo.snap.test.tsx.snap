// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Logo should render correctly 1`] = `
<div>
  <div
    class="logo"
  >
    <a
      class="logo__link"
      href="/"
    >
      <span
        class="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        class="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        class="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
</div>
`;
